data/json/martialarts.json

```diff
@@ -145,7 +145,7 @@
         "type" : "martial_art",
         "id" : "style_tai_chi",
         "name" : "Tai Chi",
-        "description" : "Though tai chi is often seen as a form of mental and physical exercise, it is a legitimate martial art, focused on self-defense.  Its ability to absorb the force of an attack makes your Perception decrease damage further on a block.",
+        "description" : "Though Tai Chi is often seen as a form of mental and physical exercise, it is a legitimate martial art, focused on self-defense.  Its ability to absorb the force of an attack makes your Perception decrease damage further on a block.",
```

Comment:
Martial arts are not proper nouns and shouldn't be capitalized

Reply:
@Leland If that is the case I might have to go back and look at the descriptions in this file. I can see a few cases of unnecessary caps already.